### DISCUSSION TITLE
How to use CPU multiple threading

### DISCUSSION THREAD

**Discussion by homink** [NVIDIA-NeMo/NeMo#8742] [Q&A]
Hi, I would like to see how fast parakeet-ctc-0.6b can perform the inference for single audio input using multiple threading on CPU. I configured such environment variables as _OMP_NUM_THREADS_ and _MKL_NUM_THREADS_ in Ubuntu but didn't work. _num_workers_ in the transcribe function didn't help neither. Any suggestions?

**Comment by titu1994** [comment]
Pytorch uses CPU threads already, so there's not much faster you can go on CPU.
